app/controllers/api/v1/instances_controller.rb

```diff
@@ -1,7 +1,12 @@
 # frozen_string_literal: true
 
 class Api::V1::InstancesController < ApiController
+  before_action :set_instance_presenter, only: [:show]
   respond_to :json
 
   def show; end
+
+  def set_instance_presenter
+    @instance_presenter = InstancePresenter.new
```

Comment:
I might just call this `@instance` here, so that from the show view perspective we have a whole bunch of `{ @instance.whatever }` lines, which reads nicely from a "show me the instance in this view" perspective.